sn_client/src/wallet.rs

```diff
@@ -59,21 +59,70 @@ impl WalletClient {
         Self { client, wallet }
     }
 
-    /// Stores the wallet to disk.
+    /// Stores the wallet to the local wallet directory.
+    /// # Example
+    /// ```no_run
+    /// # use sn_client::{Client, WalletClient, Error};
+    /// # use tempfile::TempDir;
+    /// # use bls::SecretKey;
+    /// # use sn_transfers::{LocalWallet, MainSecretKey};
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<(),Error>{
+    /// # let client = Client::new(SecretKey::random(), None, false, None, None).await?;
+    /// # let tmp_path = TempDir::new()?.path().to_owned();
+    /// # let mut wallet = LocalWallet::load_from_path(&tmp_path,Some(MainSecretKey::new(SecretKey::random())))?;
```

Comment:
I'm not really familiar with doc tests, but why do we have certain code commented out here?